packages/vats/test/bootstrapTests/test-liquidation.js

```diff
@@ -0,0 +1,592 @@
+        value: { relValue: SECONDS_PER_HOUR / 2n },
+      },
+      StartFrequency: {
+        type: 'relativeTime',
+        value: { relValue: SECONDS_PER_HOUR },
+      },
+      StartingRate: { type: 'nat', value: 10500n }, // 105%
+    },
+  });
+
+  const borrower = await walletFactoryDriver.provideSmartWallet(
+    'agoric1borrower',
+  );
+
+  await borrower.executeOfferMaker(Offers.vaults.OpenVault, {
+    offerId: 'open-vault1',
+    collateralBrandKey,
+    wantMinted: setup.vaults[0].ist,
+    giveCollateral: setup.vaults[0].atom,
+  });
+  t.like(borrower.getLatestUpdateRecord(), {
+    updated: 'offerStatus',
+    status: { id: 'open-vault1', numWantsSatisfied: 1 },
+  });
+
+  await borrower.executeOfferMaker(Offers.vaults.OpenVault, {
+    offerId: 'open-vault2',
+    collateralBrandKey,
+    wantMinted: setup.vaults[1].ist,
+    giveCollateral: setup.vaults[1].atom,
+  });
+  t.like(borrower.getLatestUpdateRecord(), {
+    updated: 'offerStatus',
+    status: { id: 'open-vault2', numWantsSatisfied: 1 },
+  });
+
+  await borrower.executeOfferMaker(Offers.vaults.OpenVault, {
+    offerId: 'open-vault3',
+    collateralBrandKey,
+    wantMinted: setup.vaults[2].ist,
+    giveCollateral: setup.vaults[2].atom,
+  });
+  t.like(borrower.getLatestUpdateRecord(), {
+    updated: 'offerStatus',
+    status: { id: 'open-vault3', numWantsSatisfied: 1 },
+  });
+
+  // Verify starting balances
+  t.like(readLatest('published.vaultFactory.managers.manager0.vaults.vault0'), {
+    debtSnapshot: { debt: { value: scale6(setup.vaults[0].debt) } },
+    locked: { value: scale6(setup.vaults[0].atom) },
+    vaultState: 'active',
+  });
+  t.like(readLatest('published.vaultFactory.managers.manager0.vaults.vault1'), {
+    debtSnapshot: { debt: { value: scale6(setup.vaults[1].debt) } },
+    locked: { value: scale6(setup.vaults[1].atom) },
+    vaultState: 'active',
+  });
+  t.like(readLatest('published.vaultFactory.managers.manager0.vaults.vault2'), {
+    debtSnapshot: { debt: { value: scale6(setup.vaults[2].debt) } },
+    locked: { value: scale6(setup.vaults[2].atom) },
+    vaultState: 'active',
+  });
+
+  const buyer = await walletFactoryDriver.provideSmartWallet('agoric1buyer');
+  {
+    // ---------------
+    //  Place bids
+    //   Bid 1: 80 IST at 10% discount
+    //   Bid 2: 90 IST at price of $9
+    //   Bid 3: 150 IST at 15% discount
+    // ---------------
+
+    const parseAmount = makeParseAmount(agoricNamesRemotes, Error);
+    await buyer.sendOffer(
+      Offers.psm.swap(
+        agoricNamesRemotes.instance['psm-IST-USDC_axl'],
+        agoricNamesRemotes.brand,
+        {
+          offerId: 'print-ist',
+          wantMinted: 1_000,
+          pair: ['IST', 'USDC_axl'],
+        },
+      ),
+    );
+
+    const maxBuy = '10000ATOM';
+
+    // bids are long-lasting offers; smartWallet.executeOffer
+    // returns a promise for (among other things) the payouts.
+    // We don't want to wait that long.
```

Comment:
```suggestion
    // We don't want to wait that long, so we use sendOfferMaker
    // rather than executeOfferMaker
```

and maybe move to the psm trade above?

Or maybe delete this comment, because presumably it's saying what sendOfferMaker does, and sendOfferMaker's docstring should do that

Reply:
cut down